### PR TITLE
[DGPFET-110] Product Array Algorithm

### PR DESCRIPTION
**Pull request by chudylobor**

### CODE REVIEW

File: submissions/godsonOgbuokiri/Product Array/arrays.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+let products = [
+    'Canon Kit Lens 15-55mm', 'Nikon 3100', 'Samsung NX300', 'Canon EOS 80D', 'Sony Alpha A7s', 'Sony NX 300M', 'Panasonic Lumix GH4', 'Panasonic Lumix DC-GH5', 'Xiaomi Y1-M1', 'Fujifilm X-A10',
+    'Canon DSLR Rebel T7', 'Canon DSLR EOS 2000D', 'Sony A7ii', 'Sony A6500 Mirrorless', 'Sony Alpha A7r III', 'Sony NEX-3N', 'Nikon V1', 'Sony NEX-F3', 'Sony NEX-5', 'Olympus PEN E-PL6',
+];
+
+let productPrice = [500, 200, 750, 2000, 1600, 300, 800, 5000, 500, 600, 1200, 1200, 2000, 2000, 5000, 750, 200, 800, 900, 750];
+
+for (let i = 0; i < products.length; i++) {
+    for (let n = 0; n < productPrice.length; n++) {
```
Comment (Mayowa2020) [inline submissions/godsonOgbuokiri/Product Array/arrays.js:9]: Well done @chudylobor ! Apparently your solution incorrect. What you should be looking is the length of the each product name not the length of the price.

File: submissions/godsonOgbuokiri/Product Array/arrays.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        if (products[i] % 2 == 0) {
+            const price = productPrice[n] * 2;
```
Comment (Mayowa2020) [inline submissions/godsonOgbuokiri/Product Array/arrays.js:11]: How are constants used in JavaScript?
JavaScript Constants

Once a constant is initialized, we cannot change its value. Simply, a constant is a type of variable whose value cannot be changed. Note: If you are sure that the value of a variable won't change throughout the program, it's recommended to use const .

File: submissions/godsonOgbuokiri/Product Array/arrays.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+            document.write('The price of ' + products[i] + ' ' + 'is ' + price + '<br>');
+        }
+        else {
+            const price = productPrice[n] / 2;
```
Comment (Mayowa2020) [inline submissions/godsonOgbuokiri/Product Array/arrays.js:15]: How are constants used in JavaScript?
JavaScript Constants

Once a constant is initialized, we cannot change its value. Simply, a constant is a type of variable whose value cannot be changed. Note: If you are sure that the value of a variable won't change throughout the program, it's recommended to use const .

**Review by Mayowa2020 — APPROVED** [review]
In this case you have used the same const Price for 2 different values, which is not allowed. Please change it to let instead. Please read through: https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference .
Kindly effect the requested changes and tag me @Mayowa2020 on GitHub or @Bukky on Slack, once you are done.

**Review by Mayowa2020 — CHANGES_REQUESTED** [review]
Please effect the request changes for your PR to be merged.

**Comment by chudylobor** [comment]
@Mayowa2020

**Review by Mayowa2020 — CHANGES_REQUESTED** [review]
Well done @chudylobor ! Please find attached and remove line 9 in the attached.
Godson

**Review by Mayowa2020 — APPROVED** [review]
Well done @chudylobor ! Approved...

**Pull request merged**